pkg/controller/controller.go

```diff
@@ -197,18 +182,12 @@ func (w *WorkflowController) sync(key string) error {
 		return err
 	}
 	glog.V(6).Infof("Syncing %s/%s", namespace, name)
-
-	obj, exists, err := w.workflowStore.GetByKey(key)
+	sharedWorkflow, err := w.WorkflowLister.Workflows(namespace).Get(name)
 	if err != nil {
-		glog.Errorf("unable to get Workflow %s/%s: %v", namespace, name, err)
+		glog.Errorf("unable to get Workflow %s/%s: %v. Maybe deleted", namespace, name, err)
+		//runtime.HandleError(fmt.Errorf("workflow '%s' in work queue no longer exists", key))
```

Comment:
Intended?

Reply:
It is. I'm going to open an issue for this.

Reply:
Here it is: #11